src/platform/site-wide/announcements/components/ClaimIncreaseBanner.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import backendServices from '../../../../platform/user/profile/constants/backendServices';
+
+export default function ClaimIncreaseBanner({ dismiss, isLoggedIn, profile }) {
+  if (profile.loading) return <div/>;
+  if (isLoggedIn && !profile.services.includes(backendServices.CLAIM_INCREASE_AVAILABLE)) return <div/>;
```

Comment:
Is it better to return an empty div or null?

Reply:
Not sure, I copied that from another announcement component.

Reply:
:man_shrugging: A'ight. Maybe something somewhere down the line expects it to be a React element or component and null throws an exception?

Either way, it's probably okay.